app/models/scheduled_event.rb

```diff
@@ -59,4 +60,12 @@ def calendar_events(start)
       end
     end
   end
-end
\ No newline at end of file
+
+  def self.day_of_month_for(event_name)
```

Comment:
Thanks for adding it. I have modified the specs as you mentioned, please do check and let me know if you have any further questions. Thanks you @brianweiner